src/views/Simulation/utils/pixiUtils.js

```diff
@@ -236,22 +296,58 @@ const createLinkGraphics = (links, setSelectedElementId, settings) => {
     };
 
     graphics.moveTo(source.x + sourceOffset.x, source.y + sourceOffset.y);
-    graphics.bezierCurveTo(
-      controlPoint1.x,
-      controlPoint1.y,
-      controlPoint2.x,
-      controlPoint2.y,
-      target.x - targetOffset.x,
-      target.y - targetOffset.y
-    );
+
+    if (link.target.type === 'stock') {
```

Comment:
Based on the mockups, I think we could use instead:
`if (link.type !== 'transports') {`

We already have the info in `link.type`, nothing to change in the graph processing functions. The existing link types are `input`, `output` and `transports`.